tests/unit/models/test_utils.py

```diff
@@ -1,13 +1,86 @@
+import sys
 from unittest import TestCase
 
+from xrpl.constants import CryptoAlgorithm
 from xrpl.models.currencies import IssuedCurrency
 from xrpl.models.exceptions import XRPLModelException
+from xrpl.models.requests import Sign
+from xrpl.models.transactions import AccountSet, AccountSetAsfFlag, Payment, PaymentFlag
+
+_ACCOUNT = "r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ"
+_FEE = "0.00001"
+_SEQUENCE = 19048
+_DOMAIN = "asjcsodafsaid0f9asdfasdf"
+_TRANSACTION = AccountSet(
+    account=_ACCOUNT,
+    fee=_FEE,
+    set_flag=AccountSetAsfFlag.ASF_DISALLOW_XRP,
+    domain=_DOMAIN,
+    sequence=_SEQUENCE,
+)
+
+_SEED = "randomsecretseedforakey"
 
 currency = "BTC"
 issuer = "r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ"
 
+_DESTINATION = "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn"
+_XRP_AMOUNT = "10000"
+
 
 class TestUtils(TestCase):
     def test_kwargs_req(self):
-        with self.assertRaises(XRPLModelException):
-            IssuedCurrency(currency, issuer)
+        if sys.version_info.major == 3 and sys.version_info.minor < 10:
```

Comment:
@mvadari  I have updated the unit tests to depend on the version of Python

Reply:
I recommend writing a helper function to do this.

Reply:
Also, for consistency (and in case they backport the changes), I'd recommend doing the same check you do in `KW_ONLY_DATACLASS`.

Reply:
good point